documentation/src/main/asciidoc/topics/ref_cache_store_sfs.adoc

```diff
@@ -17,20 +17,35 @@ is used only if an entry can fit within it. Likewise, if you remove all entries
 from memory, the single file store does not decrease in size or become
 defragmented.
 
+{brandname} stores data in the global persistent location. You can optionally
+specify a filesystem directory with the `path` attribute but the value should
+always be relative to the global persistent location.
```

Comment:
This has the same issue as the xsd description.

Reply:
Let's remove this. Info belongs with the global persistent location content.